idioms/default.md

```diff
@@ -0,0 +1,50 @@
+# The `Default` Trait
+
+## Description
+
+Many types in Rust have a [Constructor]. However, this is *specific* to the
```

Comment:
Nit: "constructor" should be lowercase here, at least.

I'm not sure if the Constructor *pattern* should be capitalized or not, but AFAIK when we talk about a constructor *function* it is always in lowercase.